src/benchq/resource_estimation/azure.py

```diff
@@ -0,0 +1,133 @@
+################################################################################
+# © Copyright 2022-2023 Zapata Computing Inc.
+################################################################################
+import os
+from collections import Counter
+from dataclasses import dataclass
+from typing import Dict, Optional
+import warnings
+
+from azure.quantum.qiskit import AzureQuantumProvider
+from orquestra.integrations.qiskit.conversions import export_to_qiskit
+from orquestra.quantum.circuits import Circuit
+from qiskit.tools.monitor import job_monitor
+
+from ..data_structures import BasicArchitectureModel, QuantumProgram
+
+
+@dataclass
+class AzureResourceInfo:
```

Comment:
suggestion (non-blocking): can't most of these variables be inherited from a superclass?

Reply:
Yes, but that's a separate story 👌